{"name": "test_add_user_incorrect_body", "status": "passed", "description": "TC003: Add User - 400 Bad request \"User validation \n    failed\" (Empty body)", "attachments": [{"name": "log", "source": "eabbaf62-7bea-42d1-9673-ff2a56a06c08-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "a57ecac3-b3d9-47a3-a0da-8b0e016f2a2a-attachment.txt", "type": "text/plain"}], "start": 1733906254607, "stop": 1733906254765, "uuid": "affa9f20-c289-4695-acd3-9697c0a71f92", "historyId": "410536b8d04ee707f137102cd761121d", "testCaseId": "410536b8d04ee707f137102cd761121d", "fullName": "API.test_api#test_add_user_incorrect_body", "labels": [{"name": "severity", "value": "minor"}, {"name": "tag", "value": "API"}, {"name": "tag", "value": "extended"}, {"name": "parentSuite", "value": "API"}, {"name": "suite", "value": "test_api"}, {"name": "host", "value": "DESKTOP-0LB7JBM"}, {"name": "thread", "value": "11428-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "API.test_api"}]}